Ventivader/Ventivader/ViewModels/EditPatientProfileViewModel.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  EditPatientProfileViewModel.swift
+//  Ventivader
+//
+//  Created by Al on 10/05/20.
+//
+
+import UIKit
+import HealthKit
```

Comment:
reverse the order so order of imports are in alphabetical order. WDYT? Also can be done in sections e.g. all Core iOS imports in alphebetical order, all other Kits , third party imports and so on.

Reply:
Good call! Done